frontend/packages/kubevirt-plugin/integration-tests/views/editBootOrderView.ts

```diff
@@ -0,0 +1,16 @@
+import { $, $$, by } from 'protractor';
+
+export const bootableDevices = (namespace: string, name: string) =>
+  $$(`#${namespace}-${name}-boot-order ol li`);
+export const bootOrderDialog = $(`[aria-label="Virtual machine boot order"]`);
+export const saveButton = bootOrderDialog.element(by.buttonText('Save'));
+export const cancelButton = bootOrderDialog.element(by.buttonText('Cancel'));
+export const addDeviceButton = bootOrderDialog.element(by.buttonText('Add source'));
+export const addDeviceSelect = bootOrderDialog.$('#add-device-select');
+export const draggablePointer = (index) =>
+  bootOrderDialog.$$(`div[style="cursor: move;"]`).get(index);
```

Comment:
@yaacov These selectors (this and the one below) are not optimal, maybe you could guide me to the code of these element so that we could add some `test-data-id` attribute?

Reply:
:+1:

https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/components/dnd-list/dnd-data-list-item.tsx#L76
https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/components/dnd-list/dnd-data-list-item.tsx#L13